src/Uno.UI.RuntimeTests/Tests/System/Given_DispatcherQueueTimer.cs

```diff
@@ -74,6 +74,36 @@ public async Task When_ScheduleRepeatingWorkItem()
 			}
 		}
 
+		[TestMethod]
+		[RunsOnUIThread]
+		public async Task When_ScheduleNonRepeatingWorkItem()
+		{
+			var tcs = new TaskCompletionSource<bool>();
+			var timer = DispatcherQueue.GetForCurrentThread().CreateTimer();
+
+			try
+			{
+				timer.IsRepeating = false;
+				timer.Interval = TimeSpan.FromMilliseconds(10);
+				timer.Tick += (snd, e) => tcs.TrySetResult(snd.IsRunning);
+
+				timer.Start();
+
+				Assert.IsTrue(timer.IsRunning);
+				Assert.IsFalse(timer.IsRepeating);
+
+				await Task.WhenAny(tcs.Task, Task.Delay(30000));
+
+				Assert.IsTrue(tcs.Task.IsCompleted);
+				Assert.IsFalse(tcs.Task.Result);
```

Comment:
Codacy found an issue: [Replace this use of 'Task.Result' with 'await'.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5943777)